service/src/main/java/uk/gov/hmcts/reform/fpl/service/ManageHearingsService.java

```diff
@@ -112,7 +112,7 @@
         List<Element<HearingBooking>> toBeReListedHearings = caseData.getToBeReListedHearings();
 
         Map<String, Object> listAndLabel = new HashMap<>(Map.of(
-            HEARING_DATE_LIST, asDynamicList(futureHearings),
+            HEARING_LIST, asDynamicList(nonCancelledHearings),
             PAST_HEARING_LIST, asDynamicList(pastAndTodayHearings),
             VACATE_HEARING_LIST, asDynamicList(nonCancelledHearings),
             TO_RE_LIST_HEARING_LIST, asDynamicList(toBeReListedHearings)
```

Comment:
Need to populate separate lists for `HEARING_DATE_LIST` and `VACATE_HEARING_LIST` for display purposes despite being identical as we cannot mix AND and OR for FieldShowConditions